src/app/statistics-page/components/daily-statistics/daily-statistics.component.ts

```diff
@@ -23,7 +31,7 @@ export class DailyStatisticsComponent implements OnInit {
     this.maxDate.setHours(0, 0, 0, 0);
 
     this.selectedDay = new Date();
-
+    
```

Comment:
Since you added property with initialized private field, this line can be removed.

Reply:
Done